Dockerfile

```diff
@@ -1,8 +1,34 @@
 FROM ubuntu:xenial
 
-RUN apt-get update && \
-    apt-get install -y curl && \
+# Install the JDK
+# We are going to need some 32 bit binaries because aapt requires it
+# file is need by the script that creates NDK toolchains
+ENV DEBIAN_FRONTEND noninteractive
+RUN dpkg --add-architecture i386 && \
+    apt-get update -qq && \
+    apt-get install -y file git curl wget zip unzip bsdmainutils strace lsof \
```

Comment:
I prefer to have one package per line and in alphabetical order. That usually helps while checking if a particular package is installed and avoiding duplicates.